services/server/server.go

```diff
@@ -469,6 +489,11 @@ func (s *Server) validateChangeSettingsRequest(ctx context.Context, req *serverp
 		return status.Error(codes.FailedPrecondition, "STT cannot be enabled because telemetry is disabled via DISABLE_TELEMETRY environment variable.")
 	}
 
+	// ignore req.EnableAlerting even if they are present since that will not change anything
+	if req.DisableAlerting && s.envSettings.EnableAlerting {
```

Comment:
@AlekSi how will our API toggle work with this? it will always fail a request with DisableAleting set to true

Reply:
If the value is set by environment variable, it can't be changed via API. That's intentional – you can't redefine process' environment, and process' environment can't be in conflict with settings. See DISABLE_TELEMETRY for the similar situation

Reply:
got it thanks